pinot-segment-local/src/main/java/org/apache/pinot/segment/local/indexsegment/mutable/MutableSegmentImpl.java

```diff
@@ -1098,6 +1200,61 @@ void updateMVEntry(int numValuesInMVEntry) {
     }
   }
 
+  static class IngestionAggregator {
```

Comment:
(minor) Don't see much value on this inner util class. Suggest removing this class and change the method name to `getMetricsAggregators()`

Reply:
done